src/encoded/static/components/genome_browser.js

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 import _ from 'underscore';
 import { FetchedData, Param } from './fetched';
 import AutocompleteBox from './region_search';
+import { filterForVisualizableFiles } from './filegallery';
 
```

Comment:
I could _not_ find anything I would call authoritative about circular dependencies with ES6 modules, but it _sounds_ like it’s acceptable. When we used CommonJS require/module.export, circular dependencies caused hideously inscrutable problems, so I stayed away from them ever since. I wanted to find something authoritative that says it’s now OK with ES6 modules or not OK, but couldn’t find anything. So I’ll just say I’d _prefer_ not having circular dependencies, but I can’t require a change here as I can’t find anything against it besides people’s opinions.

This has to do with genome_browser.js importing from filegallery.js, and filegallery.js importing from genome_browser.js.

Reply:
We can just import everything from genome_browser.js and not have circular logic.